tests/lib/openstack_query/runners/test_query_runner.py

```diff
@@ -33,14 +43,15 @@ def test_run_with_only_client_side_filter(
         mock_cloud_domain = MagicMock()
         mock_cloud_domain.name = "test"
 
-        # test with no kwargs as we test with kwargs in server-side filter
```

Comment:
Why have we removed this comment from this unit test?

Reply:
the test is `test_run_with_only_client_side_filter` so its sort of a redundant comment because its just saying that we're testing with no server-side filters